Readme.md

```diff
@@ -95,12 +95,12 @@ let package = Package(
             targets: ["MyPackage"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/Moya/Moya.git", .exact(Version(10, 0, 0, prereleaseIdentifiers: ["beta", "1"])))
+        .package(url: "https://github.com/Moya/Moya.git", .exact("10.0.0-beta.1")
```

Comment:
does it need to be exact, or can we just specify version 10, and it will choose the latest published beta/stable?

Reply:
Unfortunately in this case the resolver would say that the graph can't be resolved because he needs version starting from 10.0.0 :(